package.json

```diff
@@ -21,26 +21,27 @@
     "url": "https://github.com/Sage/carbon/issues"
   },
   "homepage": "https://github.com/Sage/carbon#readme",
+  "peerDependencies": {
+    "flux": "^3.0.0",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
   "devDependencies": {
     "babel-standalone": "~6.17.0",
     "carbon-factory": "1.1.7",
     "enzyme": "~2.8.2",
     "express": "~4.14.0",
-    "flux": "~2.1.1",
+    "flux": "^3.0.0",
```

Comment:
See above comment on peer dependencies (this is set to match whatever we go with in peer dependencies).